.github/workflows/ui-tests.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: '3.13'
```

Comment:
I'd suggest keeping stable branches on minimum supported version to avoid accidentally pushing incompatible changes.